WTWSMS/history/provinces/676 - Kakheti.txt

```diff
@@ -15,8 +15,12 @@ b_davidgareja = temple
 #b_gremi = castle
 
 # Misc
-culture = armenian
-religion = georgian_orthodox
+culture = juhuro
+religion = tzadikite
```

Comment:
You skipped my question: why do the Juhurim rule that province without consequence or strife if they were not significantly in control of that location?